target/linux/ath79/base-files/etc/board.d/02_network

```diff
@@ -76,6 +76,12 @@ ath79_setup_interfaces()
 	yuncore,a770)
 		ucidef_set_interfaces_lan_wan "eth1" "eth0"
 		;;
+        comfast,cf-wr650ac-v1|\
+        comfast,cf-wr650ac-v2)
```

Comment:
This is not resolved! Neither the execute bit nor the leading spaces.

Reply:
nope, there is no space
not sure what "bit" are you referring to

Reply:
1. Indent for both lines IS done by spaces.
2. You are changing file mode from 755 to 644. (meaning execute bit is removed as this is the only part of file permission git cares about)

Reply:
Not changing anything and spaces are not there. Not sure you look at the correct thing

Reply:
With your latest update (_below_ this comment), spaces are gone. It also looks like execute bit is correct now. So you may mark this is resolved.

Reply:
Not changing anything and spaces are not there. Not sure you look at the correct thing

Reply:
Not changing anything and spaces are not there. Not sure you look at the correct thing